staff/marc-uson/array-functions/concat.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * merge any number of arrays passed into a new array
+ * 
+ * @param {Array} array1 
+ * @param  {...Array} arrays 
+ */
+
+function concat(...arrays){
+    for(var x = 0; x < arrays.length; x++){
```

Comment:
con concat original te permite encadenar arrays y elementos simples
por ejemplo: 
concat([1,2], 3)  // [1,2,3]
puedes comprobar si es un array y si lo es añadir con un for, y si no lo es añadirlo directamente

Reply:
toda la razon, cambiado.